tests/lib/nested.sh

```diff
@@ -1288,25 +1283,25 @@ nested_start_core_vm_unit() {
             mv OVMF_VARS.ms.fd /usr/share/OVMF/OVMF_VARS.ms.fd
         fi
 
-        # In this case the kernel.efi is unsigned and signed with snaleoil certs
-        if [ "$NESTED_FORCE_MS_KEYS" != "true" ] && [ "$NESTED_BUILD_SNAPD_FROM_CURRENT" = "true" ]; then
-            OVMF_VARS="snakeoil"
-        else
-            OVMF_VARS="ms"
-        fi
-
         if nested_is_secure_boot_enabled; then
-            OVMF_CODE="secboot"
-            if os.query is-arm; then
-                cp -f "/usr/share/AAVMF/AAVMF_VARS.fd" "$NESTED_ASSETS_DIR/AAVMF_VARS.fd"
-                PARAM_BIOS="-drive file=/usr/share/AAVMF/AAVMF_CODE.fd,if=pflash,format=raw,unit=0,readonly=on -drive file=$NESTED_ASSETS_DIR/AAVMF_VARS.fd,if=pflash,format=raw"
+            OVMF_CODE=".secboot"
+            # In this case the kernel.efi is unsigned and signed with snaleoil certs
```

Comment:
This contradicts itself? The kernel cannot be both unsigned and signed.

Reply:
I did not read that comment, just moved it. But yes, it should go under the first branch of the if. The kernel gets unsigned, then resigned with snakeoil. I can just remove the comment. Not sure this is useful. Just confusing.